pkg/services/store/object/dummy/dummy_server.go

```diff
@@ -378,7 +385,7 @@ func (i *dummyObjectServer) Search(ctx context.Context, r *object.ObjectSearchRe
 // This sets the TenantId on the request GRN
 func getFullGRN(ctx context.Context, grn *object.GRN) *object.GRN {
 	if grn.TenantId == 0 {
-		modifier := store.UserFromContext(ctx)
+		modifier := appcontext.MustUser(ctx)
```

Comment:
I get nervous when I see this... panic!  but I i guess that is what happens here already